src/blocks/small-search.css

```diff
@@ -0,0 +1,45 @@
+.small-search__input {
+  font-weight: 400;
+  font-size: 14px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 33px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11;
+  background-position: 13 14;
```

Comment:
The `background-size` and `background-position` properties are missing units (e.g., 'px'). Ensure that these values have the correct units to avoid potential rendering issues.